doc/data/messages/b/bad-mcs-classmethod-argument/details.rst

```diff
@@ -1 +1,8 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+Used when a metaclass class method has a first argument named differently
+than the value specified in valid-metaclass-classmethod-first-arg option
+(default to `cls`), recommended to easily differentiate them from regular
+instance methods.
```

Comment:
This should be in the code description of bad-mcs-classmethod in the checker that will automatically end up in the documentation.